Sources/Sentry/SentryScope.m

```diff
@@ -63,6 +65,10 @@
 
 @property (atomic, strong) NSMutableArray<SentryAttachment *> *attachmentArray;
 
+@property (nullable, atomic, strong) id<SentrySpan> currentTransaction;
```

Comment:
L: it's an internal naming so it does not matter but is just `transaction` not possible? there's only one transaction here so `current` is meaningless

Reply:
No, because will generate a conflict with setTransaction method available in SentryScope, because of objc way to generate setters.

Reply:
Of course, we can use any other name other than "transaction".

Reply:
`currentTransaction` works well to be explicit, but I believe this should be `currentSpan` instead.

The fact that span is a transaction is an implementation detail